### ISSUE TITLE
Add OperationInput/OpeationOutput impl for axum-extra types

### ISSUE THREAD

**Issue by Zizico2** [tamasfe/aide#141]
Would it make sense to add some implementations for `axum-extra` here? Under the `axum-extra` flag? My particular use case is the `Either*` extractors

**Comment by jplatte** [comment]
Yes, PRs welcome. Anything that's feature-flagged in axum-extra should be feature-flagged here too, see existing features.
